### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by dreamsayer** [aniruddha-adhikary/mrt-buddy#43] [General]
May I suggest to give the app a better look with a good looking icon?

![IMG_20241106_011352](https://github.com/user-attachments/assets/0d600977-071f-48e5-9903-7e7866d2b902)

You can use this one if you want. 

Also is there a way to add an update checker within the app?

**Comment by Zamiul-rashid** [comment]
@aniruddha-adhikary should I add this one when I'm updating the readme file as well?
